docs/pages/versions/unversioned/sdk/imagepicker.md

```diff
@@ -173,6 +173,7 @@ Display the system UI for taking a photo with the camera. Requires `Permissions.
 
   Option for videos:
 
+  - **videoMaxDuration (_number_)** -- Maximum duration, in seconds, for video recording. Setting this to 0 disables the limit. Defaults to 0 (no limit). **On Android**, effect of this option depends on support of installed camera app. **On Web** this option has no effect - the limit is browser-dependant.
```

Comment:
Well, you have to convince me more about this `0` default value that should be recognized as `no-limit` option.
I'm a bit skeptical here .. 🧐

Reply:
Maybe better not to mention about `0` so people don't try to use it explicitly and only treat it as no limit (for null or undefined `doubleValue` in Obj-C would return `0.0` either way)

Reply:
Not specifying any value on iOS would result in 10-minutes limited recording: https://developer.apple.com/documentation/uikit/uiimagepickercontroller/1619146-videomaximumduration 😅 
Dunno about Android 😉

Reply:
I have reproduced current behavior - on iOS not specifying `UIImagePickerController.videoMaximumDuration` defaults it to 600 (this is the exact reason for #9482).

On Android, however, there's no standardised default limit unless you set it explicitly as intent extra. How the limit value is handled and what is its default - depends on selected camera app. Android default cam app handles it as expected - when I put the limit to intent - it stops recording, and without that intent, I gave up after ~15 minutes ;)

As `0` value seems a bit hacky - we can leave it undocumented value and just set platform defaults (see below).